src/main.rs

```diff
@@ -41,6 +41,18 @@ enum Command {
         package: PackageId,
     },
 
+    /// Packages this api
+    #[clap(alias = "pack")]
+    Package {
+        /// Destination directory for the release
+        #[clap(long)]
+        #[arg(default_value = ".")]
+        directory: String,
+        /// Generate package but do not write it to filesystem
+        #[clap(long)]
+        dry_run: bool,
```

Comment:
I would say this is not needed – people can just delete the tgz

Reply:
I think this is nice to have if you just want to validate the manifest and don't want to risk overwriting something. Unless you feel it's not worth the complexity I'd prefer to keep it.